src/microbe_stage/Membrane.cs

```diff
@@ -201,6 +210,32 @@ public bool Contains(float x, float y)
         return crosses;
     }
 
+    public void EnableEngulfAnimation(bool enable, double delta)
```

Comment:
```suggestion
    public void HandleEngulfAnimation(bool enable, double delta)
```

I'd say that is a better more clear name.